### PR TITLE
UserGuide.adoc: Update clear and import features.

### PR DESCRIPTION
**Pull request by Exeexe93**
User Guide.adoc: Update clear and import features.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -425,10 +479,18 @@ ot/ORDER_TYPE,tid/TRANSACTION_ID, n/NAME, a/ADDRESS, p/PHONE_NUMBER, e/EMAIL, rt
 * Duplicate and invalid order or return order will not be imported into Delino.
 ====
 
-==== Examples
+[[import-combination]]
+There are only one possible combination for the `import` Command:
+
+. Possible combination of Import command
```
Comment (JeremyLoh) [inline docs/UserGuide.adoc]: Need to remove the space for `. Possible combination of Import command`, or else the table name will not be shown.
Should be `.Possible combination of Import command` instead

image

**Review by JeremyLoh — DISMISSED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Exeexe93) [reply docs/UserGuide.adoc]: Done! Thanks for the look out!

**Review by JeremyLoh — DISMISSED** [review]
Travis build failed because the svg file (ArchitectureSequenceDiagram) added has no newline at EOF.
image

The Ui image for the main page is also not working
image

https://deploy-preview-226--ay1920s2-cs2103t-w12-1.netlify.com/index.html

**Review by JeremyLoh — APPROVED** [review]
LGTM

**Review by Cherweijie — APPROVED** [review]
LGTM to test for rendering of png files.

**Pull request merged**